c7n/resources/health.py

```diff
@@ -78,10 +78,23 @@ def resources(self, query=None):
             query['filter'] = q
         return super(HealthEvents, self).resources(query=query)
 
+    def augment(self, resources):
+        client = local_session(self.session_factory).client('health')
+        for r in resources:
+            affectedEntities = client.describe_affected_entities(
```

Comment:
don't think we want to augment affected entities if the eventTypeCategory isn't specific to an entity (like accountNotification for example).